### PR TITLE
Add comments for Semi truck fields

### PR DESCRIPTION
**Pull request by aaronpkahn**
# Description

Documents the Semi truck only fields.

## Type of change
- [ X ] Documentation update

# Checklist:
- [X ] I have made corresponding updates to the documentation.

### CODE REVIEW

File: protos/python/vehicle_data_pb2.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: vehicle_data.proto
-# Protobuf Python Version: 4.25.0
```
Comment (agbpatro) [inline protos/python/vehicle_data_pb2.py:4]: did you generate with some other version of protobuf?

Comment (aaronpkahn) [reply protos/python/vehicle_data_pb2.py:4]: yeah, probably need to update my version

File: protos/python/vehicle_metric_pb2.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: vehicle_metric.proto
-# Protobuf Python Version: 4.25.0
+# Protobuf Python Version: 4.25.1
```
Comment (agbpatro) [inline protos/python/vehicle_metric_pb2.py:4]: you would need to update https://github.com/teslamotors/fleet-telemetry/blob/main/.github/workflows/build.yml#L22 and also readme

Comment (aaronpkahn) [reply protos/python/vehicle_metric_pb2.py:4]: updated

**Review by agbpatro — APPROVED** [review]

**Pull request merged**